modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -251,7 +252,7 @@ export default class Ecdsa {
       shares.nShares[P_j.j] = {
         i: P_j.j,
         j: P_i.i,
-        n: P_j.n,
+        n: P_i.n,
```

Comment:
bug in key derive, passing the wrong modulus value.

Reply:
We had tests testing signing with derived keys, which seemed to work, so this didnt seem to matter before? 

But for paillier proofs, we need the modulus (the n value), so tests started to fail because the incorrect n value was passed around. 

Paired with @islamaminBitGo on this bug.